pyls/_utils.py

```diff
@@ -152,6 +149,17 @@ def clip_column(column, lines, line_number):
     return min(column, max_column)
 
 
+def position_to_jedi_linecolumn(document, position):
+    """Convert the format 'line', 'character' to 'line', 'column'"""
+    code_position = {}
+    if position:
```

Comment:
@bnavigator what are the possible values for position?

Reply:
PyLS's wideline used position dict for the LSP protocol
``python {'line': int, 'character': int}``

https://microsoft.github.io/language-server-protocol/specification#position